### ISSUE TITLE
csi defaults to C#7

### ISSUE THREAD

**Issue by marek-safar** [dotnet/roslyn#21327]
**Version Used**: 2.3.1.61919

**Steps to Reproduce**:

run csi.exe
```
Microsoft (R) Visual C# Interactive Compiler version 2.3.1.61919
Copyright (C) Microsoft Corporation. All rights reserved.

Type "#help" for more information.
> int i = default;
(1,9): error CS8107: Feature 'default literal' is not available in C# 7. Please use language version 7.1 or greater.
```

**Expected Behavior**:

Default to latest C# or allow to set it

**Actual Behavior**:

No C# 7.1 features in csi.exe

**Comment by jcouv** [comment]
@marek-safar Thanks for pointing this out. I'll look into it.

**Comment by xCyborg** [comment]
Any updates on this? I still can't find how to use the latest language features in csi or C# interactive.

**Comment by jcouv** [comment]
This was fixed in 15.5 (not sure exactly which preview) which is why I closed the issue.
PR: https://github.com/dotnet/roslyn/pull/21331

**Comment by jcouv** [comment]
Verified this works:

```
>csi
Microsoft (R) Visual C# Interactive Compiler version 2.6.0.62308
Copyright (C) Microsoft Corporation. All rights reserved.

Type "#help" for more information.
> int i = default;
```
